omnivoice/cli/demo.py

```
@@ -235,115 +242,109 @@ def _gen_core(
-
-    with gr.Blocks(theme=theme, css=css, title="OmniVoice Demo") as demo:
-        gr.Markdown(
-            """
-# OmniVoice Demo
-
-State-of-the-art text-to-speech model for **600+ languages**, supporting:
-
-- **Voice Clone** — Clone any voice from a reference audio
-- **Voice Design** — Create custom voices with speaker attributes
+            nsr = gr.Checkbox(
+                label=_("Normalize Before Silence Removal"),
```

Ruff (0.16.2)

[error] Undefined name `_`

(F821)